src/xdocs/config_misc.xml

```diff
@@ -302,21 +302,21 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         <p>Example #2: Comment is placed at the end of the block and has previous statement.</p>
         <source>
 1   public void foo1() {
-2       foo2();
-3       // it is OK
+2     foo2();
+3     // it is OK
```

Comment:
@romani Why do we have line numbers on these examples?
Should they be removed in another issue?

Reply:
@rnveach I think keeping the line numbers is helpful to a certain extent. It helps in describing the error or warnings regarding the indentation in the code.

Reply:
@varunu28 This may be true, but I don't remember any of our other examples having line numbers.
Also none of the descriptions reference the line numbers, so it is kinda pointless.

Reply:
@rnveach Done

Reply:
We do use line positions inside the code for some comments, but I didn't see any that couldn't just be swapped with `above` or `below` wording.

Reply:
It is better to remove line numbers